.travis.yml

```diff
@@ -27,25 +27,31 @@ matrix:
     env: PUPPET_VERSION="~> 5.0" CHECK=build DEPLOY_TO_FORGE=yes
   - rvm: 2.4.3
     bundler_args: --without development release
-    dist: trusty
+    dist: bionic
```

Comment:
This is from https://github.com/voxpupuli/modulesync_config so if we're going to switch, it needs to happen there.